.github/workflows/setup-env.yml

```diff
@@ -0,0 +1,33 @@
+name: Setup env file contents
```

Comment:
New workflow file that sets sup vars for env file. This allows less duplication of code.